fanficfare/adapters/tests/conftest.py

```diff
@@ -0,0 +1,669 @@
+import pytest
```

Comment:
Should this file be named something specific to chiread?  Or is the intention that all adapters with similar tests will all share one big file?

Reply:
`conftest.py` is loaded by pytest when launching tests, I don't think I can rename it.
But youre right : it's a big file, we may create wile for each adapter. I try this.